test/protocol-buffer-conversion-test.ts

```diff
@@ -1,6 +1,28 @@
 import { assert } from 'chai'
 import XRPCurrency from '../src/XRP/xrp-currency'
+import XRPPathElement from '../src/XRP/xrp-path-element'
+import XRPPath from '../src/XRP/xrp-path'
 import { Currency } from '../src/generated/web/org/xrpl/rpc/v1/amount_pb'
+import { Payment } from '../src/generated/web/org/xrpl/rpc/v1/transaction_pb'
+import { AccountAddress } from '../src/generated/web/org/xrpl/rpc/v1/account_pb'
+
+// TODO(amiecorso): Refactor tests to separate files.
+// Set up global variables for use in test cases
+const testCurrencyProto: Currency = new Currency()
+testCurrencyProto.setCode(new Uint8Array([1, 2, 3]))
+testCurrencyProto.setName('currencyName')
+
+const testAccountAddress = new AccountAddress()
+testAccountAddress.setAddress('r123')
+
+const testAccountAddressIssuer = new AccountAddress()
+testAccountAddressIssuer.setAddress('r456')
```

Comment:
Should you be able to set the address to an invalid address?

Reply:
This is a protocol buffer so it is just typed as 'string' as an underlying data type and thus any string is valid. This is just test data so we're just making our lives easy and abusing this privilege.